manifests/charts/fleet-manager/templates/deployment.yaml

```diff
@@ -22,10 +22,23 @@ spec:
         image: {{ .Values.image.hub }}/fleet-manager:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         name: manager
+        ports:
+          - containerPort: 9443
```

Comment:
why 9443

Reply:
I am not sure how the code allows customizing  `webhook-port` while in this manifest it is hardcoding. Can this work?

Reply:
yes, it has been tested and it works. I refer this: https://github.com/kurator-dev/kurator/blob/1f80323adece56664984c048309b560c178baff1/manifests/charts/cluster-operator/templates/deployment.yaml#L38-L40

Reply:
Not very right, if you specify a different via flag